.github/workflows/release.yml

```diff
@@ -106,8 +106,23 @@ jobs:
             ${{ matrix.runtime }}-info.txt
           if-no-files-found: error
 
+  release-binary:
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    steps:
+      - uses: actions/checkout@v4
+      - run: sudo apt-get install -y protobuf-compiler
+      - run: cargo build-bajun
+        env:
+          SKIP_WASM_BUILD: 1
```

Comment:
We want the statically linked wasm to be there, otherwise our local chains can't run.

Reply:
I have adjusted the workflow to also build the WASM binary